contracts/collections/sg721-base/src/contract.rs

```diff
@@ -54,7 +56,8 @@ where
         self.parent.contract_info.save(deps.storage, &info)?;
 
         let minter = deps.api.addr_validate(&msg.minter)?;
-        self.parent.minter.save(deps.storage, &minter)?;
+        self.minter.save(deps.storage, &minter)?;
```

Comment:
this will need to go away and use cw_owner only.

Reply:
This has been addressed by using ```        let owner = cw_ownable::get_ownership(deps.storage);```